study.md

```diff
@@ -50,15 +52,15 @@ In your own words, define what the responsibilities of the controller layer are
 in Rails.
 
 ```md
-<!-- your answer here -->
+controllers take url of the user - the input and call model methods and pass outputs of it to the view
```

Comment:
The controller doesn't take and URLs from a user.  The router takes URLs and verbs to determine which controller action to execute.